infer_subc/organelles/cellmask.py

```diff
@@ -117,6 +123,187 @@ def choose_max_label_cellmask_union_nucleus(cellmask_img: np.ndarray,
 
     return cellmask_out > 0
 
+def rescale_intensity(in_img: np.ndarray):
```

Comment:
Why scale from 0-10 instead of using a 0-1 min-max, like min_max_intensity_normalization func. from infer_subc.core.img?

Reply:
removed edits in cellmask.py